src/core/Window.h

```diff
@@ -58,39 +58,39 @@ class Window {
     Window() = default;
 
   private:
-    static bool KeyDown(int32_t dwScancode);
-    static bool KeyUp(int32_t dwScancode);
+    static bool KeyDown(int32_t scancode);
+    static bool KeyUp(int32_t scancode);
     static void AllKeysUp(void);
-    static void OnFullscreenChanged(bool bIsNowFullscreen);
+    static void OnFullscreenChanged(bool isNowFullscreen);
```

Comment:
this goes with the comment in `Window.cpp` about this name, just `isFullscreen` or `fullscreen` seems better to me